clover_simulation/launch/simulator.launch

```diff
@@ -3,7 +3,7 @@
     <arg name="mav_id" default="0"/>
     <arg name="est" default="ekf2"/> <!-- PX4 estimator: lpe, ekf2 -->
     <arg name="vehicle" default="clover"/> <!-- PX4 vehicle configuration: clover, clover_vpe -->
-    <arg name="main_camera" default="true"/> <!-- Simulated vision position estimation camera (optical flow, ArUco) -->
+    <arg name="main_camera" default="false"/> <!-- Simulated vision position estimation camera (optical flow, ArUco) -->
```

Comment:
This change will break the Clover image and Clover simulator by default. If you need to disable the camera, then it should be done in a way without changing the original repo.